src/ucp/api/ucp.h

```diff
@@ -889,6 +904,12 @@ typedef struct ucp_ep_params {
      */
     ucs_sock_addr_t         sockaddr;
 
+    /**
+     * Address of a remote endpoint directly connect to.
```

Comment:
1. I think that need to clarify the wording here.
2. Does this field come with a corresponding bit in field_mask?
3. Is ep_addr used only when calling ucp_ep_create from the server side?